docs/source/user-guide/installation.md

```diff
@@ -57,13 +57,17 @@ Note that the project builds for Scala 2.12 by default but can be built for Scal
 make release PROFILES="-Pspark-3.4 -Pscala-2.13"
 ```
 
-## Run Spark with Comet enabled
+## Run Spark Shell with Comet enabled
 
 Make sure `SPARK_HOME` points to the same Spark version as Comet was built for.
 
 ```console
+export COMET_JAR=spark/target/comet-spark-spark3.4_2.12-0.1.0-SNAPSHOT.jar
+
 $SPARK_HOME/bin/spark-shell \
-    --jars spark/target/comet-spark-spark3.4_2.12-0.1.0-SNAPSHOT.jar \
+    --jars $COMET_JAR \
+    --conf spark.driver.extraClassPath=$COMET_JAR \
```

Comment:
do we need extraClassPath? --jars adds to both driver and executor 

https://spark.apache.org/docs/latest/configuration.html
spark.jars		Comma-separated list of jars to include on the driver and executor classpaths. Globs are allowed.

Reply:
Yes, we need to specify extraClassPath to resolve the `ClassNotFoundException` issues described in https://github.com/apache/datafusion-comet/issues/182. This is specific to `spark-shell` and not needed for `spark-submit`.

Reply:
It seems some difference on handling extra classpaths on the driver and the executors between spark-submit and spark-shell.